src/DataSets/newWhoStandardDataSets/ZScores/ChartDataZscores.ts

```diff
@@ -0,0 +1,208 @@
+        datasetValues: lhfa_g_2_5_y_z,
+        metadata: {
+          label: ChartCodes.lhfa_g_2_5_y_z.label,
+          measurementType: measurementTypeCodes.lh_cm,
+          timeUnit: timeUnitCodes.months,
+          range: { start: 24, end: 60 }
+        }
+      }
+    }
+  },
+  [CategoryCodes.wfa_b]: {
+    datasets: {
+      [ChartCodes.wfa_b_0_5_y_z]: {
+        datasetValues: wfa_b_0_5_y_z,
+        metadata: {
+          label: ChartCodes.wfa_b_0_5_y_z.label,
+          measurementType: measurementTypeCodes.w_kg,
+          timeUnit: timeUnitCodes.months,
+          range: { start: 0, end: 60 }
+        }
+      },
+      [ChartCodes.wfa_b_0_13_w_z]: {
+        datasetValues: wfa_b_0_13_w_z,
+        metadata: {
+          label: ChartCodes.wfa_b_0_13_w_z.label,
+          measurementType: measurementTypeCodes.w_kg,
+          timeUnit: timeUnitCodes.weeks,
+          range: { start: 0, end: 13 }
+        }
+      }
+    }
+  },
+  [CategoryCodes.wfa_g]: {
+    datasets: {
+      [ChartCodes.wfa_g_0_5_y_z]: {
+        datasetValues: wfa_g_0_5_y_z,
+        metadata: {
+          label: ChartCodes.wfa_g_0_5_y_z.label,
+          measurementType: measurementTypeCodes.w_kg,
+          timeUnit: timeUnitCodes.months,
+          range: { start: 0, end: 60 }
+        }
+      },
+      [ChartCodes.wfa_g_0_13_w_z]: {
+        datasetValues: wfa_g_0_13_w_z,
+        metadata: {
+          label: ChartCodes.wfa_g_0_13_w_z.label,
+          measurementType: measurementTypeCodes.w_kg,
+          timeUnit: timeUnitCodes.weeks,
+          range: { start: 0, end: 13 }
+        }
+      }
+    }
+  },
+  [CategoryCodes.wflh_b]: {
+    datasets: {
+      [ChartCodes.wfh_b_2_5_y_z]: {
+        datasetValues: wfh_b_2_5_y_z,
+        metadata: {
+          label: ChartCodes.wfh_b_2_5_y_z.label,
+          measurementType: measurementTypeCodes.w_kg,
+          timeUnit: timeUnitCodes.months,
+          range: { start: 65, end: 120 }
+        }
+      },
+      [ChartCodes.wfl_b_0_2_y_z]: {
+        datasetValues: wfl_b_0_2_y_z,
+        metadata: {
+          label: ChartCodes.wfl_b_0_2_y_z.label,
+          measurementType: measurementTypeCodes.w_kg,
+          timeUnit: timeUnitCodes.months,
+          range: { start: 45, end: 110 }
+        }
+      }
+    }
+  },
+  [CategoryCodes.wflh_g]: {
+    datasets: {
+      [ChartCodes.wfh_g_2_5_y_z]: {
+        datasetValues: wfh_g_2_5_y_z,
+        metadata: {
+          label: ChartCodes.wfh_g_2_5_y_z.label,
+          measurementType: measurementTypeCodes.w_kg,
+          timeUnit: timeUnitCodes.months,
+          range: { start: 65, end: 120 }
+        }
+      },
+      [ChartCodes.wfl_g_0_2_y_z]: {
+        datasetValues: wfl_g_0_2_y_z,
+        metadata: {
+          label: ChartCodes.wfl_g_0_2_y_z.label,
+          measurementType: measurementTypeCodes.w_kg,
+          timeUnit: timeUnitCodes.months,
+          range: { start: 45, end: 110 }
+        }
+      }
+    }
+  }
```

Comment:
Can you take a look at the range. The range itself may be correct but the interval in the datasetValues are 0.5 and not 1, which leads to the graph only mapping through half the file

Reply:
Thank you!